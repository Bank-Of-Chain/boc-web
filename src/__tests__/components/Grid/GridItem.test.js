import React from "react";
import { render } from "@testing-library/react";
import GridItem from "../../../components/Grid/GridItem";

test("GridItem Component Render", () => {
  const { asFragment } = render(<GridItem />);
  expect(asFragment()).toMatchSnapshot();
});
