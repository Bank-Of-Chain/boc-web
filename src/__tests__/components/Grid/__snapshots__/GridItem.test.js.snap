// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridItem Component Render 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root makeStyles-grid-1  MuiGrid-item"
  />
</DocumentFragment>
`;
